pkg/util/arrays.go

```diff
@@ -15,12 +15,26 @@
 package util
 
 import (
+	"bytes"
+	"crypto/md5"
 	"fmt"
 	"strings"
 )
 
 const keyPairSeparator = "="
 
+// GenerateMD5Hash will generate a MD5 hash from the given map
+func GenerateMD5Hash(source map[string]string) string {
```

Comment:
what is the purpose of this?

Reply:
To compare the content between the image protobuf files and the configMap files:
https://github.com/kiegroup/kogito-cloud-operator/pull/83/files#diff-a35a3b1905a737e2fc36cb08fac9a195R70